packages/pixeloven-react/testing/package.json

```diff
@@ -34,7 +33,8 @@
         "react-dom": "16.13.1"
     },
     "peerDependencies": {
-        "@pixeloven-react/common": "6.x"
```

Comment:
This was an accident. As common requires testing for well testing... but the opposite is not true.